CONTRIBUTING.md

```diff
@@ -61,6 +61,12 @@ _Before_ submitting a pull request, please make sure the following is done…
     $ yarn test
     ```
 
+#### Testing with your own project
+
+You can use `yarn run release-from-git` to create releases as tags on github. This requires that:
+ - Your git remote (where you want to publish the tags) is `origin`
+ - Your commit messages follow the conventional commits specification
```

Comment:
this is not quite clear. A minimal example would help a lot.

Reply:
I added a minimal example and a link to the spec